packages/ListBox/stories/examples/Keyboard/index.js

```diff
@@ -0,0 +1,333 @@
+import React from "react";
+import Blocks, { Block } from "storybook/components/Blocks";
+
+import ListBox from "../../../src";
+import CardComponent from "../../../../Card/src";
+
+import * as characters from "../../fixtures/characters";
+import Toast from "../../../../Toast/src";
+
+const Card = ({ children }) => <CardComponent style={{ padding: "16px", width: "280px" }}>{children}</CardComponent>;
+const CheckListItem = ({ children }) => {
+  /* eslint-disable jsx-a11y/label-has-associated-control */
+  return (
+    <div style={{ padding: "8px", display: "flex", position: "relative" }}>
+      <label>
+        <div style={{ position: "absolute", top: "10px", left: "-20px" }}>
+          <input type="checkbox" />
```

Comment:
To facilitate manual keyboard testing, what about putting `tabindex="-1"` on this?